app/frontend/src/components/HorizontalScroller.tsx

```diff
@@ -13,7 +13,6 @@ const useStyles = makeStyles((theme) => ({
     display: "inline-flex",
     flexDirection: "row",
     height: "100%",
-    width: "100%",
```

Comment:
scroll snap stops working with this.
I think you have to leave this in and do `"& > *": { flexShrink: 0 }`

Reply:
@Jesus82 just needs the flex shrink 0 to fix storybook